### PR TITLE
add json outpuy

### PR DESCRIPTION
**Pull request by sante85**

### CODE REVIEW

**Comment by sante85** [comment]
@rosswarren can evaluate?

File: epevermodbus/command_line.py [hunk 1]
```diff
@@ -56,112 +57,178 @@ def main():
...
+        output["low_voltage_disconnect_voltage"] = controller.get_low_voltage_disconnect_voltage()
+        output["discharging_limit_voltage"] = controller.get_discharging_limit_voltage())
+        output["battery_rated_voltage"] = controller.get_battery_rated_voltage()
+        output["default_load_on_off_in_manual_mode"] = controller.get_default_load_on_off_in_manual_mode()
+        output["equalize_duration"] = controller.get_equalize_duration()
+        output["boost_duration"] = controller.get_boost_duration()
+        output["battery_discharge"] = controller.get_battery_discharge()
+        output["battery_charge"] = controller.get_battery_charge()
+        output["charging_mode"] = controller.get_charging_mode()
+        print(output)
```
Comment (SmithChart) [inline epevermodbus/command_line.py:124]: Please use the builtin ``json.dumps(output)`` here.
Using a library we make sure that the json will always be well-formatted and can be read across platforms.
The Python ``str()`` output of a dict should not be considered stable.

**Comment by sante85** [comment]
pr updated

**Comment by SmithChart** [comment]
@sante85 Thx! LGTM

**Comment by sante85** [comment]
when you publish on pypi.org?

thanks

Il giorno dom 24 dic 2023 alle ore 13:42 SmithChart <
***@***.***> ha scritto:

**Comment by sante85** [comment]
hi Rosswarren/Epevermodbus,
when you think to publish?

thanks

Il giorno dom 24 dic 2023 alle ore 17:03 Sante Stanisci ***@***.***>
ha scritto:

**Comment by sante85** [comment]
@SmithChart  @rosswarren @hrford you are here to merge PR? thanks

**Comment by SmithChart** [comment]
@sante85 Sorry, I am just a contributor and can not merge.

**Comment by sante85** [comment]
Why the owner not merge?

Il ven 5 gen 2024, 19:20 SmithChart ***@***.***> ha scritto:

**Comment by sante85** [comment]
i have update, @rosswarren @rafael2k  @hrford please merge

**Comment by rosswarren** [comment]
thank you, these changes have been incorporated into https://github.com/rosswarren/epevermodbus/pull/26 so closing this PR

**Pull request closed without merging**